src/pages/_proxied-dot-io/consul/home/style.module.css

```diff
@@ -30,3 +30,21 @@
 		margin-bottom: 128px;
 	}
 }
+
+/**
+ * Hero
+ */
+
+.heroActionsPrimary {
+	display: grid;
+
+	& > * {
+		grid-area: 1 / 1;
+		color: transparent;
+	}
+
+	&.settled > *:not([aria-hidden='true']) {
+		color: white;
+		z-index: 1;
+	}
+}
```

Comment:
Stack both buttons directly on top of each other making each width equal to the greatest button width.